scipy-1.0/paper.tex

```diff
@@ -662,10 +662,10 @@ \subsection*{Project scope}
 PyStan\cite{pystan-ref} cover Bayesian statistics and probabilistic modeling.
 scikit-image\cite{vanderwalt2014scikit} provides image processing
 capabilities beyond SciPy's \texttt{ndimage}, SymPy\cite{meurer2017sympy}
-provides a Python interface for symbolic computation, and
-while \texttt{sparse.csgraph} or \texttt{spatial} offers basic tools
-for working with graphs and networks compared to a more specialized
-Python library like NetworkX\cite{hagberg2008networkx}.
+provides a Python interface for symbolic computation, and \texttt{sparse.csgraph}
+and \texttt{spatial} offer basic tools
+for working with graphs and networks compared to more specialized
```

Comment:
I think `more` can be omitted, it is already covered by `specialized`. Apart from that LGTM.